scenarios/gaia_tasks/dataset.py

```
@@ -1,127 +1,112 @@
-        ]
-    }
-]
+def get_gaia_tasks(limit: int = None) -> List[Dict]:
+    """Get GAIA tasks directly from Hugging Face, with fallback to local tasks."""
+    try:
+        loader = HuggingFaceDatasetLoader()
+        dataset = loader.load_benchmark_dataset('gaia', limit=limit)
+        return dataset['tasks']
+    except Exception as e:
+        print(f"⚠️  Could not load GAIA from HuggingFace ({e}), using local fallback tasks")
+        return get_local_gaia_tasks(limit)
```

Ruff (0.12.2)

Undefined name `get_local_gaia_tasks`

(F821)